deploy/README.md

```diff
@@ -68,8 +68,8 @@ The following matrix displays the tested package versions for our Helm chart.
 
 Sumo Logic Helm Chart | kube-prometheus-stack/Prometheus Operator | FluentD | Fluent Bit | Falco | Metrics Server | Telegraf Operator | Tailing Sidecar Operator
 |:-------- |:-------- |:-------- |:-------- |:-------- |:-------- |:--------  |:--------
-2.1.1 - Latest | 12.3.0 | 1.12.1 | 0.15.4 | 1.7.10 | 5.8.4 | 1.1.5 | 0.3.0
-2.1.0 | 12.3.0 | 1.12.1 | 0.15.1 | 1.7.10 | 5.8.1 | 1.1.5 | 0.3.0
```

Comment:
2.1.0 has the `0.15.1`
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.1.0/deploy/helm/sumologic/Chart.yaml#L16